src/pages/Home/Home.tsx

```diff
@@ -29,38 +30,45 @@ export const Home: FC = () => {
   const [detailTx, setDetailTx] = useState<Transaction | null>(null);
   const [transactions, setTransactions] = useState<Transaction[]>([]);
 
-  const { logout } = appCtx;
+  const theme = darkMode ? "dark" : "light";
 
   useEffect(() => {
-    instance
-      .get("/lightning/list-all-tx?reversed=true")
-      .then((tx: any) => {
-        setTransactions(tx.data);
-      })
-      .catch((_) => {
-        logout();
-      });
-  }, [logout]);
-
-  const showSendModalHandler = () => {
-    setShowSendModal(true);
-  };
-
-  const closeSendModalHandler = (confirmed?: boolean) => {
-    setShowSendModal(false);
-    if (confirmed) {
-      const theme = appCtx.darkMode ? "dark" : "light";
-      toast.success(t("tx.sent"), { theme });
+    if (!walletLocked) {
+      instance
+        .get("/lightning/list-all-tx?reversed=true")
+        .then((tx: any) => {
```

Comment:
This `any` is intentional I guess. Wanna add a comment for this?

Reply:
Not intentional, thank you :+1: 
I usually use `any` at first to not break the flow, but sometimes I forget to add it :D

Reply:
What we do in projects is to create an `any`-alias called `FixMe` which indicates, that this type needs to be defined later on. While `any` would mean it's intentional (and could/should be commented why).